src/components/BookingModal/BookingModalForm/index.js

```diff
@@ -44,19 +49,17 @@ const BookingModalForm = props => {
     if (isEventBeingUpdated) {
       return [
         {
-          label: booking.messages
-            ? 'Submit'
-            : `Update to ${
-              bookingDuration === 0.5 ? 'Half' : bookingDuration
-            } ${bookingDuration > 1 ? 'Days' : 'Day'}`,
+          label: isEventCancelled ? 'Cancelled' : rejectionMessages ? 'Submit' : `Update to ${
```

Comment:
Nested ternaries are very hard to read, can you split this out into a function call or something.